common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +456,199 @@ def test_teams_membership(self):
+                  'EdX was founded by the Massachusetts Institute of Technology and Harvard '
+                  'University in May 2012. EdX was created for students and institutions that seek '
+                  'to transform themselves through cutting-edge technologies, innovative pedagogy, and '
+                  'rigorous courses. More than 70 schools, nonprofits, corporations, and international '
+                  'organizations offer or plan to offer courses on the edX website. As of 22 October 2014, '
+                  'edX has more than 4 million users taking more than 500 courses online.'
+        )
+
+        self.assertEqual(
+            self.create_team_page.validation_message_text,
+            'Your team could not be created. Check the highlighted fields below and try again.'
+        )
+
+    def test_user_can_create_new_team_successfully(self):
+        """
+        Scenario: The user should be able to create new team.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Create Team page for that topic
+        Then I should see the Create Team header and form
+        When I fill all the fields present with appropriate data
+        And I click Create button
+        Then I should see teams list page with newly created team.
+        """
+        self.assertEqual(self.browse_teams_page.get_pagination_header_text(), 'Showing 0 out of 0 total')
+        self.verify_and_navigate_to_create_team_page()
+
+        self.fill_create_form()
+        self.create_team_page.create_team()
+
+        self.create_team_page.wait_for_ajax()
+        self.browse_teams_page.is_browser_on_page()
+
+        self.assertEqual(self.browse_teams_page.get_pagination_header_text(), 'Showing 1 out of 1 total')
+        # Verify the newly created team content.
+        team_card = self.browse_teams_page.team_cards.results[0]
+        self.assertEqual(team_card.find_element_by_css_selector('.card-title').text, 'Avengers')
+        self.assertEqual(
+            team_card.find_element_by_css_selector('.card-description').text,
+            'The Avengers are a fictional team of superheroes.'
+        )
+
+    def test_user_can_cancel_the_team_creation(self):
+        """
+        Scenario: The user should be able to cancel the creation of new team.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Create Team page for that topic
+        Then I should see the Create Team header and form
+        When I click Cancel button
+        Then I should see teams list page without any new team.
+        """
+        self.assertEqual(self.browse_teams_page.get_pagination_header_text(), 'Showing 0 out of 0 total')
+
+        self.verify_and_navigate_to_create_team_page()
+        self.create_team_page.cancel_team()
+
+        self.create_team_page.wait_for_ajax()
```

Comment:
Nit: add the `wait_for_ajax()` call to the `cancel_team()` method. That way it's not the caller's responsibility to prevent flaky test failures.